hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/has/HasCallAttempt.java

```diff
@@ -50,6 +51,9 @@ public class HasCallAttempt extends AbstractCallAttempt<HasCallAttempt> {
     @Nullable
     private final Account redirectAccount;
 
+    @NonNull
+    private final SignatureVerifier signatureVerifier;
```

Comment:
Is there a possibility that this could be useful in the base class?

Reply:
I'll move it in there.  I already wanted to (but didn't) put a `CustomGasCalculator` in there next to the `SystemContractGasCalculator`.  On my next go-around.

Reply:
To be fixed #16448

Reply:
NVM.  Started on this.  _Lots_ of tests needed to be changed.  I had changed 22 files before stopping, wasn't nearly done.  Should reconsider how we're using `{Has,Hts}CallAttempt` in unit tests.